From 28ed42b7ccb68911fd6e9b8c163b076f689fbc30 Mon Sep 17 00:00:00 2001
From: Filippe LeMarchand <gasinvein@gmail.com>
Date: Thu, 28 Feb 2019 18:29:20 +0300
Subject: [PATCH] Update git-version.cmake

---
 rpcs3/git-version.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rpcs3/git-version.cmake b/rpcs3/git-version.cmake
index 087a3c1bda..7662c02dc1 100644
--- a/rpcs3/git-version.cmake
+++ b/rpcs3/git-version.cmake
@@ -25,7 +25,7 @@ if(GIT_FOUND AND EXISTS "${CMAKE_SOURCE_DIR}/.git/")
 	if(NOT ${exit_code} EQUAL 0)
 		message(WARNING "git rev-parse failed, unable to include git branch.")
 	endif()
-	execute_process(COMMAND ${GIT_EXECUTABLE} describe --tags --abbrev=0
+	execute_process(COMMAND ${GIT_EXECUTABLE} describe --always --tags --abbrev=0
 		WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
 		RESULT_VARIABLE exit_code
 		OUTPUT_VARIABLE RPCS3_GIT_TAG)